descriptors/ModuleDescriptor-template.json

```diff
@@ -375,16 +376,37 @@
       "handlers": [
         {
           "methods": [
-            "GET", "HEAD"
+            "GET",
+            "HEAD"
           ],
           "pathPattern": "/reservoir/upload-form*",
           "permissionsRequired": []
         }
       ]
+    },
+    {
+      "id": "reservoir-sru",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/reservoir/sru",
+          "permissionsRequired": [
+            "reservoir-sru.get"
```

Comment:
I don't think we should be putting any permissions on the /sru endpoint, it makes it inaccessible to  standard clients. We don't require any permissions for the OAI endpoint anyway.

Reply:
OK